src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const marriedPrefix = sex === 'm' ? ' is my wife' : ' is my husband';
```

Comment:
it's actually `suffix` to the name, you can figure out better names here than prefix and suuffix